### PR TITLE
Improve failover test case

### PR DESCRIPTION
**Pull request by VijithaEkanayake**
## Purpose
remove unused codes and clean up the test.

## Approach
Use three failover endpoints and remove the unnecessary counter variables.

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: tests/ballerina-integration-test/src/test/resources/resiliency/resiliencyservices/http_failover_05.bal [hunk 1]
```diff
@@ -209,67 +106,8 @@ service mock05 bind backendEP05 {
         path: "/"
     }
     mockResource(endpoint caller, http:Request req) {
-        counter05++;
-        if (counter05 % 5 == 0) {
-            runtime:sleep(30000);
-        }
         http:Response response = new;
-        if (req.hasHeader(mime:CONTENT_TYPE)
```
Comment (arukshani) [inline tests/ballerina-integration-test/src/test/resources/resiliency/resiliencyservices/http_failover_05.bal]: What is the reason for the removal of this test case? Is there an alternative for this test?

Comment (VijithaEkanayake) [reply tests/ballerina-integration-test/src/test/resources/resiliency/resiliencyservices/http_failover_05.bal]: Yes, We do have separate bal files for different tests to improve the tests execution time. For this particular test, we do not need this logic as those are redundant. 

Multipart tests[1] are not removed from the resiliency tests but the required service is in a separate bal file[2]. Just removed the duplicate codes which not required for this test.

[1]. https://github.com/ballerina-platform/ballerina-lang/blob/master/tests/ballerina-integration-test/src/test/java/org/ballerinalang/test/service/resiliency/HttpResiliencyTest.java#L86

[2]. https://github.com/ballerina-platform/ballerina-lang/blob/master/tests/ballerina-integration-test/src/test/resources/resiliency/resiliencyservices/http_failover_01.bal#L217

File: tests/ballerina-integration-test/src/test/resources/resiliency/resiliencyservices/http_failover_05.bal [hunk 2]
```diff
@@ -209,67 +106,8 @@ service mock05 bind backendEP05 {
...
-    errorStatusResource(endpoint caller, http:Request req) {
-        http:Response outResponse = new;
-        outResponse.statusCode = 503;
-        outResponse.setPayload("Failure status code scenario");
-        caller->respond(outResponse) but {
-            error e => log:printError(
-                           "Error sending response from mock service", err = e)
-        };
+        response.setPayload("Mock Resource is Invoked.");
+        caller->respond(response) but {error e => log:printError("Error sending response from mock service", err = e)};
```
Comment (chamil321) [inline tests/ballerina-integration-test/src/test/resources/resiliency/resiliencyservices/http_failover_05.bal]: Shall we directly send the payload as response is not involved with any other operations??
caller->respond("Mock Resource is Invoked.") but {error e => log:printError("Error sending response from mock service", err = e)};

File: tests/ballerina-integration-test/src/test/resources/resiliency/resiliencyservices/http_failover_05.bal [hunk 3]
```diff
@@ -178,27 +78,24 @@ service<http:Service> failoverDemoService05 bind failoverEP05 {
...
     }
-    echoResource(endpoint caller, http:Request req) {
+    delayResource(endpoint caller, http:Request req) {
         http:Response outResponse = new;
         // Delay the response for 30000 milliseconds to mimic network level delays.
         runtime:sleep(30000);
-
-        outResponse.setPayload("echo Resource is invoked");
+        outResponse.setPayload("Delayed resource is invoked");
         caller->respond(outResponse) but {
```
Comment (chamil321) [inline tests/ballerina-integration-test/src/test/resources/resiliency/resiliencyservices/http_failover_05.bal]: Here also we can directly set payload to respond.

**Pull request merged**